packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -19,32 +17,16 @@ interface HdsAppSideNavSignature {
     onDesktopViewportChange?: (arg: boolean) => void;
   };
   Blocks: {
-    header?: [
-      {
-        Header?: HdsAppSideNavBaseSignature['Blocks']['header'];
-        isMinimized?: boolean;
-      },
-    ];
-    body?: [
-      {
-        Body?: HdsAppSideNavBaseSignature['Blocks']['body'];
-        isMinimized?: boolean;
-      },
-    ];
-    footer?: [
-      {
-        Footer?: HdsAppSideNavBaseSignature['Blocks']['footer'];
-        isMinimized?: boolean;
-      },
-    ];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    default?: any;
   };
-  Element: HdsAppSideNavBaseSignature['Element'];
+  Element: HTMLElement;
```

Comment:
[suggestion] slightly more accurate
```suggestion
  Element: HTMLDivElement;
```